src/subsplit_dag.cpp

```diff
@@ -254,11 +252,17 @@ EigenVectorXd SubsplitDAG::BuildUniformOnAllTopologiesPrior() const {
   for (const auto &[parent_child_id, gpcsp_idx] : dag_edges_) {
     const auto &[parent_id, child_id] = parent_child_id;
     size_t child0_taxon_count, child1_taxon_count;
-    if (parent_id == child_id) {
+    if (parent_id == child_id) {  // #273
       child0_taxon_count = GetDagNode(child_id)->GetBitset().SubsplitChunk(0).Count();
       child1_taxon_count = static_cast<size_t>(taxon_count_ - child0_taxon_count);
     } else {
-      // 0 and 1 are flipped to maintain existing behavior
+      // We define "child0" as the clade of a child subsplit which has a bitset with
```

Comment:
This is GREAT. However, I feel like the meat of it belongs in the PCSP description in `bitset.hpp`, and we can refer to that here.